release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -675,6 +676,31 @@ In the table below, features are marked with the following statuses:
 |GA
 |GA
 
+|Pod-level bonding for secondary networks
+|TP
+|GA
+|GA
+
+|External DNS Operator
+|TP
+|TP
+|TP
+
+|AWS Load Balancer Operator
+|-
+|TP
+|TP
+
+|====
+
+[id="ocp-storage-technology-preview-features"]
+=== Storage technology preview features
```

Comment:
Can you do a sweep and make sure that all instances of "Technology Preview" are capitalized properly?